### PR TITLE
Continuum -> Anaconda rename

### PR DESCRIPTION
**Pull request by seibert**

### CODE REVIEW

File: docs/source/developer/contributing.rst [hunk 1]
```diff
@@ -191,7 +191,7 @@ You can expect this to take less than 20 minutes.
 
 Some platforms (such as Windows) cannot be hosted by Travis-CI, and the
 Numba team has therefore access to a separate platform provided by
-`Continuum <http://continuum.io>`_, our sponsor. We hope parts of that
+`Anaconda, Inc. <https://anaconda.com/>`_, our sponsor. We hope parts of that
```
Comment (stuartarchibald) [inline docs/source/developer/contributing.rst]: No trailing slash needed in URL.

File: docs/source/user/faq.rst [hunk 2]
```diff
@@ -99,14 +99,20 @@ Does Numba vectorize array computations (SIMD)?
 Numba doesn't implement such optimizations by itself, but it lets LLVM
 apply them.
 
-Does Numba parallelize code?
-----------------------------
+Does Numba automatically parallelize code?
+------------------------------------------
+
+In can, in some cases:
```
Comment (stuartarchibald) [inline docs/source/user/faq.rst]: s/In/It/

File: docs/source/user/faq.rst [hunk 3]
```diff
@@ -99,14 +99,20 @@ Does Numba vectorize array computations (SIMD)?
...
+
+* Ufuncs and gufuncs with the ``target="parallel"`` option will run on multiple threads.
+* The experimental ``parallel=True`` option to ``@jit`` will attempt to optimize
+  array operations and run them in parallel.
 
-No, it doesn't.  If you want to run computations concurrently on multiple
-threads (by :ref:`releasing the GIL <jit-nogil>`) or processes, you'll
-have to handle the pooling and synchronisation yourself.
+You can also manually run computations on multiple threads yourself and use
```
Comment (stuartarchibald) [inline docs/source/user/faq.rst]: Mention `prange`?

File: docs/source/user/installing.rst [hunk 4]
```diff
@@ -21,9 +21,9 @@ Installing using Conda
 
 The easiest way to install numba and get updates is by using Conda,
 a cross-platform package manager and software distribution maintained
-by Continuum Analytics.  You can either use `Anaconda
-<http://continuum.io/downloads.html>`_ to get the full stack in one download,
-or `Miniconda <http://conda.pydata.org/miniconda.html>`_ which will install
+by Anaconda, Inc.  You can either use `Anaconda
+<https://www.anaconda.com/download/>`_ to get the full stack in one download,
```
Comment (stuartarchibald) [inline docs/source/user/installing.rst]: Trailing slash.

File: README.rst [hunk 5]
```diff
@@ -35,7 +35,7 @@ Installing
 ==========
 
 The easiest way to install numba and get updates is by using the Anaconda
-Distribution: https://www.continuum.io/downloads
+Distribution: https://www.anaconda.com/downloads
```
Comment (stuartarchibald) [inline README.rst]: Think it's s/downloads/download/ the +s is a 301.

File: docs/source/user/faq.rst (hunk 3 above)
Comment (seibert) [reply docs/source/user/faq.rst]: Fixed

**Comment by stuartarchibald** [comment]
```bash
$ grep -ir "continuum"
buildscripts/incremental/install_miniconda.sh:  wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
buildscripts/incremental/install_miniconda.sh:  wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh
buildscripts/condarecipe.jenkins/custom_condarc:- http://repo.continuum.io/pkgs/free
buildscripts/condarecipe.jenkins/custom_condarc:- http://repo.continuum.io/pkgs/pro
buildscripts/condarecipe.hsa/custom_condarc:- http://repo.continuum.io/pkgs/free
buildscripts/condarecipe.hsa/custom_condarc:- http://repo.continuum.io/pkgs/pro
numba/cuda/tests/cudapy/test_vectorize_decor.py:        https://github.com/ContinuumIO/numbapro/issues/90
setup.py:    author_email="numba-users@continuum.io",
examples/notebooks/numba.ipynb:    "Have questions?  Join the mailing list at numba-users@continuum.io"
README.rst:Join the numba mailing list numba-users@continuum.io:
README.rst:https://groups.google.com/a/continuum.io/d/forum/numba-users
README.rst:See if our sponsor can help you (which can help this project): http://www.continuum.io
CONTRIBUTING.md:(https://groups.google.com/a/continuum.io/forum/#!forum/numba-users).
docs/source/developer/contributing.rst:We have a public mailing-list that you can e-mail at numba-users@continuum.io.
docs/source/developer/contributing.rst:`Google Groups <https://groups.google.com/a/continuum.io/forum/#!forum/numba-users>`_,
docs/source/developer/contributing.rst:   Python 3.4.2 |Continuum Analytics, Inc.| (default, Oct 21 2014, 17:16:37)
docs/source/conf.py:   u'Continuum Analytics', 'manual'),
docs/source/conf.py:     ['Continuum Analytics'], 1)
docs/source/conf.py:   'Continuum Analytics', 'Numba', 'One line description of project.',
docs/source/user/faq.rst:.. _NumbaPro: http://docs.continuum.io/numbapro/
docs/source/user/installing.rst:   Python 3.4.2 |Continuum Analytics, Inc.| (default, Oct 21 2014, 17:16:37)
```

I think the build scripts/mailing lists need to continue to refer to `continuum.io` and the github URL needs to stay, but the rest can probably be changed?

**Comment by seibert** [comment]
repo.continuum.io hasn't moved, and I didn't want to edit output until Conda updates the Python banner.  However, the diff notes that I forgot to actually remove the NumbaPro footnote (which was the inspiration for the FAQ fix).  I think the conf.py changes are also needed.

**Comment by stuartarchibald** [comment]
Think the bottom of `README.rst` still needs changing?
`See if our sponsor can help you (which can help this project): http://www.continuum.io` https://github.com/seibert/numba/blame/9a82d54d1b8509954b1945e32ca7037ef5bd313a/README.rst#L131
Otherwise I think this is now all done.

**Comment by stuartarchibald** [comment]
Looks good. Thanks for the patch.

**Pull request merged**